fourier/fourier_example_test.go

```diff
@@ -0,0 +1,187 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
```

Comment:
To be honest, I don't understand the image example, though I'm sure it's just unfamiliarity with the domain. I do think it would be useful to have a simple FFT of a sine wave or something to show the peak.

Reply:
Sure, I can add that, though that is essentially what the examples on the types show - these sequence is a sum of periodic functions in both of those.